.github/workflows/release-vame-package.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build and Release
+
+on:
+  workflow_run:
+    workflows: [Testing]
+    branches: [main]
+    types:
+      - completed
```

Comment:
I don't think this guarantees the tests ran successfully, it might be that the workflow can be completed even if tests failed. Can you double check this @vinicvaz ?